tests/Spec.hs

```diff
@@ -682,14 +697,32 @@ main = hspec $ do
 
       fromJust (Project.lookupPullRequest prId state') `shouldSatisfy`
         (\pr -> Project.approval pr == Just (Approval (Username "deckard") Project.MergeAndTag 0))
+    
+    it "rejects 'merge' commands to a branch other than master" $ do
+      let
+        prId = PullRequestId 1
+        state = singlePullRequestState prId (Branch "p") (Branch "m") (Sha "abc1234") "tyrell"
+
+        event = CommentAdded prId "deckard" "@bot merge"
+
+        (state', actions) = runAction $ handleEventTest event state
+
+      actions `shouldBe`
+        [ AIsReviewer "deckard"
+        , ALeaveComment prId "Pull request refused: the target branch must be the integration branch."
```

Comment:
Nit: it's not so much the _pull request_ that is being refused here, but _merge_